routes/users.js

```diff
@@ -29,4 +29,7 @@
 router.patch("/:userId", authenticate, authorizeRoles([SUPERUSER]), users.updateUser);
 router.get("/suggestedUsers/:skillId", authenticate, authorizeRoles([SUPERUSER]), users.getSuggestedUsers);
 
+// WARNING!! - One time Script/Route to do migration.
+router.post("/migrate", authenticate, authorizeRoles([SUPERUSER]), users.migrate);
```

Comment:
Nit:
Why this check failure is coming? Can you please explain

Reply:
Because RDS doesn't have a Rate Limitation - In order to implement rate limiting, the system typically uses algorithms that track the number of requests being made over a specified period of time. If the number of requests exceeds a certain threshold, the system will either reject further requests or delay them until the rate of requests falls back within the allowed limit.

Rate limiting is commonly used in web applications and APIs to prevent abusive behavior such as denial-of-service attacks or scraping of sensitive data. It can also be used to ensure fair access to resources for all users, by limiting the number of requests that any one user can make within a certain time period.